yaml/semgrep/multi-line-message.yaml

```diff
@@ -0,0 +1,18 @@
+rules:
+- id: multi-line-message
+  message: >-
+    This rule has a multi-line message field, which may display poorly
+    in a terminal. Consider ensuring it is on one line. For example,
+    use `message: >-`, not `message: |`.
+  languages: [yaml]
+  patterns:
+  - pattern-inside: 'rules: [..., $RULE, ...]'
+  - pattern: |
+      message: "=~/[\\n\\r]/"
```

Comment:
I couldn't work out the right escaping to express this as a metavariable-regex, so I've gone with the deprecated string matching syntax instead.